tox.ini

```diff
@@ -44,10 +44,11 @@ deps =
     # of key dependencies.
     # Astropy is installed from the nightly wheels
     devdeps: astropy>=0.0dev0
-    devdeps: matplotlib>=0.0dev0
-    devdeps: git+https://github.com/sunpy/sunpy
-    devdeps: git+https://github.com/sunpy/mpl-animators
     devdeps: git+https://github.com/spacetelescope/gwcs
+    devdeps: git+https://github.com/sunpy/mpl-animators
+    devdeps: git+https://github.com/sunpy/sunpy
+    devdeps: matplotlib>=0.0dev0
+    devdeps: numpy>=0.0.dev0
```

Comment:
why must you do these things :stuck_out_tongue:

Reply:
I just sorted it in order of letter

Reply:
It makes it marginally less likely to conflict with the template if we keep the things that are in the template at the top. This is true for things like the [pyproject.toml](https://github.com/sunpy/streamtracer/pull/156/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711L54-R57) as well.

Reply:
The template should be in order as well